### DISCUSSION TITLE
How to disable static rendering?

### DISCUSSION THREAD

**Discussion by mrclay** [vercel/next.js#17980] [Help]
Our site will be quite large once complete and each page needs to make ~10 API calls, so we'd like Next to skip rendering pages when building pull request branches. We can live with full rebuilds in our main integration branches.

Either skip rendering or some more limited test like making sure `tsc` is happy.

Any ideas?

**Comment by balazsorban44** [comment]
Would [on-demand static rendering](https://nextjs.org/docs/basic-features/data-fetching#incremental-static-regeneration) help?
